cdap-cli/src/main/java/co/cask/cdap/cli/CLIMain.java

```diff
@@ -91,6 +94,14 @@
   public static final Option DEBUG_OPTION = new Option(
     "d", "debug", false, "Print exception stack traces.");
 
+  @VisibleForTesting
+  public static final Option SCRIPT_FILE_OPTION = new Option(
+    "f", "scriptfile", true, "Name of script file to read and execute.  " +
+    "Blank lines and comments starting with '#' are ignored.  " + 
```

Comment:
Can we simply have anything after a `#` be ignored in all cases? I think it would be confusing if you can do `# sdfk` as a comment but not `list apps # lists apps`.